docs/ContribOperators.md

```diff
@@ -3716,9 +3716,9 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dt><tt>char_embedding_size</tt> : int</dt>
 <dd>Integer representing the embedding vector size for each char.If not provide, use the char embedding size of embedding vector.</dd>
 <dt><tt>conv_window_size</tt> : int</dt>
-<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernal shape.</dd>
+<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernel shape.</dd>
 <dt><tt>embedding_size</tt> : int</dt>
-<dd>Integer representing the embedding vector size for each word.If not provide, use the fileter size of conv weight</dd>
+<dd>Integer representing the embedding vector size for each word.If not provide, use the filter size of conv weight</dd>
```

Comment:
The source is here:
https://github.com/microsoft/onnxruntime/blob/dc486d146b125db780fa2b71eda16875d01cd5e0/onnxruntime/core/graph/contrib_ops/contrib_defs.cc#L1759